library/src/test/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapterTest.java

```diff
@@ -841,6 +841,159 @@ public void notifyItemMovedInSectionUsingSection_withAdapterWithManySections_cal
         verify(spySectionedRecyclerViewAdapter).callSuperNotifyItemMoved(11, 15);
     }
 
+    @Test
+    public void notifyNotLoadedStateChangedUsingSection_withAdapterWithManySections_callsNotifyItemChanged() {
```

Comment:
please modify these tests to test the method with tag, so in consequence the method with section will also be tested 👍